flink-libraries/flink-table/src/main/scala/org/apache/flink/table/runtime/functions/ScalarFunctions.scala

```diff
@@ -212,4 +213,9 @@ object ScalarFunctions {
     */
   def toBase64(base: String): String = Base64.encodeBase64String(base.getBytes())
 
+  /**
+    * Returns the result string which trimmed the left spaces from the base string.
+    */
+  def lTrim(base: String) = StringUtils.stripStart(base, " ")
```

Comment:
Remove this function since there is one already in calcite. We can use it directly.
Recovery this file.